docs/reference/aggregations/bucket/range-field-note.asciidoc

```diff
@@ -0,0 +1,173 @@
+[[search-aggregations-bucket-range-field-note]]
+=== Note on Bucketing Range Fields
```

Comment:
Hmm, how about "Subtleties of aggregating Range Fields" or something similar?  More of a topic and less of an addendum note vibe?

Reply:
+1 for 'Aggregating range fields' or something similar.

Also please use sentence case for titles and headings.